src/config/SettingsDataEntries.tsx

```diff
@@ -284,22 +248,56 @@ export const ConfigurableChannels: Channels.Channel[] = [
 ];
 
 export const ConfigurableLoggingChannels: Channels.Channel[] = [
-	{ name: 'Message Logs', description: 'The channel for (non-NSFW) message logs', key: 'channelsLogsMessage' },
 	{
-		name: 'Member Logs',
-		description: 'The channel for member logs, once enabled, I will post all member related events there.',
-		key: 'channelsLogsMember'
+		name: 'Member Add Logs',
+		description: 'The channel I will use to notify when a member joins.',
```

Comment:
Update this and the other descriptions here to match the descriptions in the JSON file in Skyra, after applying my suggestions there.